### PR TITLE
update hypershift test frequency

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by heliubj18** [comment]
LGTM

**Comment by YuLi517** [comment]
lgtm

**Comment by LiangquanLi930** [comment]
@liangxia could you help to review, when you have time? thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 1]
```diff
@@ -326,8 +326,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
-- as: aws-ipi-ovn-hypershift-guest-p2-f7
-  cron: 0 6 */7 * *
+- as: aws-ipi-ovn-hypershift-guest-p3-f21
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:329]: p1 = GA feature,
p2 = tech preview
p3 = dev preview
Is hypershift in dev preview ?  Perhaps you want `p1-f21` ?

Comment (YuLi517) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:329]: Currently, there is no GA for hypershift now. HyperShift is still in TP and DP.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:329]: Ack

**Comment by liangxia** [comment]
Since it's only a change of the test frequency via cron, no need to rehearse the jobs.

/pj-rehearse skip

**Pull request merged**